### ISSUE TITLE
Link[href] validation: resolve CSS/JS directory structure dependency from statics

### ISSUE THREAD

**Issue by Palmaswell** [sinnerschrader/schlump#55]
The current link validation from `validator.js` expects that the link resources are all located in the same statics directory. It would be a nice to have if **Schlump** would allow resources like CSS and JS to be located somewhere else.

If I do not explicitly redefine the path of my statics directory with the `--dest-statics` flag the validation method will throw an error. Still I have not solve the problem of my CSS/JS being dependent to the statics directory.

**Comment by KnisterPeter** [comment]
What is your use case for this? Where do you want to place them? Remember this is a static site generator and the dist folder of schlump should be the document-root. Therefore everything need to be placed somewhere in the dist folder. Schump should be able to handle this case well currently
(I mean to have the css/js somewhere in dist).

**Comment by Palmaswell** [comment]
Thanks for the quick reply @KnisterPeter :)
Sure, I expect too for all build assets to be on the `./dist` directory. Nevertheless, I do not expect for my CSS/JS to be necessarily in `./dist/statics/css/*.css` speak `./dist/statics/js/*.js`. For example I am looking  forward to place fonts fonts and images in statics `./dist/statics/images/` and CSS/JS directly in root `./dist/app.css`.

I think currently this is not possible, all CSS/JS must be at the same directory level as the statics directory. Am I right? If so, it is more than clear that  this would be a nice feature and not an issue.

**Comment by KnisterPeter** [comment]
That should already work as you described it here.
The base for validation is just the destination folder (by default `dest`).

**Comment by Palmaswell** [comment]
Sorry if I am not being clear enough. 
But currently, if I generate my CSS in  `./dist/app.css` and I do not set the statics flag to  ` --dest-statics=./dist` the link[href] validator throws an error. This happens exactly on line 74 of the **valdator.js**. And if I set the flag in this manner all of my other static assets must be flat on root dist, which is not nice.

**Comment by KnisterPeter** [comment]
You are right, this is caused by the validator now knowing 'non-static' resources.
It will be tricky to fix this in a sane manner. I need to think about possible solutions. For now you may have to stick with resources in the static folder or disable validation.

**Comment by Palmaswell** [comment]
Thanks for your time @KnisterPeter and merry Christmas!!!!!
